src/components/header/header.jsx

```diff
@@ -1,28 +1,46 @@
 import React from 'react';
-import {HEADER_TYPE} from '@types';
+import {Link} from 'react-router-dom';
+import {useSelector} from 'react-redux';
+import {getUser, getAuthorizationStatus} from '@selectors/user';
+import {AppRoute, AuthorizationStatus} from '@const';
 
-const Header = ({isAuthorized = false}) => {
-  const profile = isAuthorized
-    ? <span className="header__user-name user__name">Oliver.conner@gmail.com</span>
+const Header = () => {
+  const user = useSelector(getUser);
+  const isAuth = useSelector(getAuthorizationStatus) === AuthorizationStatus.AUTH;
+
+  const profile = isAuth
+    ? <span className="header__user-name user__name">{user.email}</span>
     : <span className="header__login">Sign in</span>;
 
+  const avatarStyle = isAuth
+    ? {backgroundImage: `url(${user.avatarUrl})`, borderRadius: `50%`}
+    : undefined;
+
   return (
     <header className="header">
       <div className="container">
         <div className="header__wrapper">
           <div className="header__left">
-            <a className="header__logo-link header__logo-link--active">
+            <Link
+              className="header__logo-link header__logo-link--active"
+              to={AppRoute.ROOT}
+            >
               <img className="header__logo" src="img/logo.svg" alt="6 cities logo" width="81" height="41" />
-            </a>
+            </Link>
           </div>
           <nav className="header__nav">
             <ul className="header__nav-list">
               <li className="header__nav-item user">
-                <a className="header__nav-link header__nav-link--profile" href="/login">
-                  <div className="header__avatar-wrapper user__avatar-wrapper">
+                <Link
+                  className="header__nav-link header__nav-link--profile"
+                  to={isAuth ? AppRoute.FAVORITES : AppRoute.LOGIN}
+                >
+                  <div className="header__avatar-wrapper user__avatar-wrapper"
+                    style={avatarStyle}
+                  >
```

Comment:
Чуть форматирование не консистетное
```suggestion
                  <div 
                    className="header__avatar-wrapper user__avatar-wrapper"
                    style={avatarStyle}
                  >
```